src/coreclr/nativeaot/Runtime/windows/PalRedhawkCommon.cpp

```diff
@@ -172,7 +172,7 @@ REDHAWK_PALEXPORT int32_t PalGetModuleFileName(_Out_ const TCHAR** pModuleNameOu
     return 0;
 }
 
-REDHAWK_PALEXPORT uint64_t __cdecl PalGetTickCount64()
+REDHAWK_PALEXPORT uint64_t REDHAWK_PALAPI PalGetTickCount64()
```

Comment:
PalGetTickCount64 is defined as cdecl in C#. It has to match.

Reply:
You can add new `RhpGetTickCount64` that is cdecl and call that from C# instead of PalGetTickCount64. And then this change can stay.

Reply:
So you propose declare `RhpGetTickCount64` in c++, for which I would add corresponding C# function, and use that function instead of `PalGetTickCount64`? I probably can just define `__cdecl` to nothing for `TARGET_X86` in that file then.

Reply:
Yes. There is number of methods like that (e.g. `RhpReleaseCastCacheLock`). They are using `__cdecl` already.